archive-cleaner

```diff
@@ -24,6 +25,14 @@ SOURCE_WHITELIST = ['core', 'extra', 'staging', 'testing', 'community', 'communi
 DEFAULT_KEEP_YEARS = 3
 
 
+def human_filesize(size):
+    for unit in ['', 'KB', 'MB', 'GB', 'TB']:
+        if size < 1024:
+            break
+        size /= 1024
+    return f'{size:.0f} {unit}'
```

Comment:
I'd use `.2f` for the format here so that we get slightly more meaningful values if we are only just over the size change (for example 1.5GB vs 1GB).